AppleStoreProject/AppleStoreProject/ForYouViewController.swift

```diff
@@ -7,4 +7,275 @@
 
 import UIKit
 
-final class ForYouViewController: UIViewController {}
+final class ForYouViewController: UIViewController {
+
+    //MARK: - Private Properties
+    private let forYouUnderLineView = UIView()
+    private let userImageButton = UIButton()
+    private let someThingNewLable = UILabel()
+    private let orderView = UIView()
+    private let orderLineView = UIView()
+    private let orderImage = UIImageView()
+    private let orderStatusImage = UIImageView()
+    private let orderStatusLable = UILabel()
+    private let orderStatusInformationLable = UILabel()
+    private let orderStatusButton = UIButton()
+    private let orderProgressView = UIProgressView()
+    private let orderProcessedLable = UILabel()
+    private let orderSendLable = UILabel()
+    private let orderDeliveredLable = UILabel()
+    private let recomendationLable = UILabel()
+    private let informationDeliveryImage = UIImageView()
+    private let informationDeliveryLable = UILabel()
+    private let notificationDeliveryLable = UILabel()
+    private let notificationDeliveryImage = UIImageView()
```

Comment:
```suggestion
    private let notificationDeliveryImageView = UIImageView()
```
это же не Image а ImageView